southworkscom/botframework-solutions
# [TypeScript][BotBuilder Libs] Solved issues

## Purpose
The current implementation of `TSLint` will be deprecated (palantir/tslint#4534).
Update `TSLint` to `ESLint` the following projects:

- Botbuilder-Solutions
- Botbuilder-Skill

## Changes
- Add `ESLint`
- Configure `ESLint` with the `TSLint` rules.
- Fix the new linting issues

![image](https://user-images.githubusercontent.com/40918752/60356399-376d2a00-99a7-11e9-83b7-aded741b3358.png)

## Testing Steps
1. Go to  `.\lib\typescript`
1. Open the console
1. Execute `rush install` to install dependencies
1. Execute `rush build`
1. Execute `rush test`
1. Verify that the test run successfully
